.travis.yml

```diff
@@ -36,9 +36,6 @@ matrix:
       os: linux
       dist: trusty
       env: LDIST=DO_NOT_DEPLOY
-    - compiler: "gcc"
```

Comment:
for this change it would be a good idea to add a small comment about why it's being removed (i.e. 12.04 has been deprecated, and will be dropped by Travis-CI in the future, etc, etc, etc)

Reply:
Added some details in the commit message and the PR description.